docs/configuration/configuration.md

```diff
@@ -267,6 +267,7 @@ The following meta labels are available on targets during relabeling:
 * `__meta_azure_machine_resource_group`: the machine's resource group
 * `__meta_azure_machine_tag_<tagname>`: each tag value of the machine
 * `__meta_azure_machine_scale_set`: the name of the scale set which the vm is part of (this value is only set if you are using a [scale set](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/))
+* `__meta_azure_machine_power_state`: the current power state of the machine
```

Comment:
small nit but it would be nice if the list were sorted.

Reply:
Sure thing :)